### DISCUSSION TITLE
Filtering does not work on a smartphone

### DISCUSSION THREAD

**Discussion by Sventovit50** [AdguardTeam/AdGuardHome#7066] [Q&A]
Hello everyone
I have AdGuard Home installed on a Raspberry and there is something that I don't understand: on machines equipped with Windows and Chrome (but it's the same with Edge) the filtering works.
If I go to canyoublockit.com the ads do not appear.
But on the smartphone (Android 14, Chrome) while the DNS points to the Raspberry (like on Windows) the filtering does not work.
Any idea?

**Comment by tescophil** [comment]
Most likely you have the Private DNS setting turned on in chrome which will bypass your AdGuard Home instance.

Settings -> Privacy and Security -> Use Private DNS
